go-1.8.7/src/runtime/os_freebsd_ppc64x.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build freebsd
+// +build ppc64 ppc64le
+
+package runtime
+
+import (
+	"runtime/internal/sys"
+)
+
+const (
+	_AT_NULL   = 0  // End of vector
+	_AT_PAGESZ = 6  // System physical page size
+	_AT_HWCAP  = 16 // hardware capability bit vector
```

Comment:
sys/elf_common.h #define        AT_HWCAP        25      /* CPU feature flags. */
sys/elf_common.h #define        AT_HWCAP2       26      /* CPU feature flags 2. */

AT_RANDOM doesn't exist.

Reply:
Lets just remove AT_RANDOM and AT_HWCAP2, they’re no used by FBSD.

Reply:
AT_HWCAP2 is used on FreeBSD (for power8)